Views/footer.phtml

```diff
@@ -0,0 +1,7 @@
+
+
+<div id="footer">

```

Comment:
Make general index.phtml template file which will declare html document structure and wil call header, content and footer separately.

Take a look inside my project i sent you.